src/3rdparty/walkontable/src/renderer/table.js

```diff
@@ -276,9 +276,9 @@ export default class TableRenderer {
 
         if (rowHeight) {
           // Decrease height. 1 pixel will be "replaced" by 1px border top
-          TR.firstChild.style.height = `${rowHeight - 1}px`;
+          TR.style.height = `${rowHeight - 1}px`;
```

Comment:
```suggestion
          TR.style.height = `${rowHeight}px`;
```

Maybe removing `1px` will help? There should be done further investigation.